ansible_events_ui/db/migrations/versions/202208231142_46df413363d4_activation_post_changes.py

```diff
@@ -0,0 +1,88 @@
+"""activation post changes
+
+Revision ID: 46df413363d4
+Revises: 4c2b57da4a58
+Create Date: 2022-08-23 11:42:40.402436+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '46df413363d4'
+down_revision = '4c2b57da4a58'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Make sure you delete these comments after the migration is manually verified and edited accordingly if needed.